codecov_auth/migrations/0037_sync_user_terms_agreement.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.2.2 on 2023-08-30 20:30
+
+from django.db import migrations
+
+
+def sync_agreements(apps, schema):
+    owners = apps.get_model("codecov_auth", "Owner")
```

Comment:
Is there a benefit of doing it this way over importing the Owner model like `from codecov_auth.models import Owner` and getting rid of this line?

Reply:
I just did it to keep it consistent with other migrations files are retrieving models in `codecov_auth`, not sure about other apps